packages/api-tests/test/window.core.spec.js

```diff
@@ -681,6 +681,23 @@ describe('WindowCore API', function(done) {
       return chainPromises(steps);
     });
 
+    it('Should call the success callback when getting the current window object #ssf.Window.getCurrentWindow', function() {
+      const getCurrentWindow = () => {
+        const script = (callback) => {
+          ssf.Window.getCurrentWindow(() => callback(true));
+        };
+        return executeAsyncJavascript(app.client, script);
+      };
+
+      const steps = [
+        () => getCurrentWindow(),
+        () => getCurrentWindow(), // Make sure the callback is called twice
```

Comment:
The problem happened when get current window was called a second time, as it was being called correctly when the window was initially created, but not when retrieving the cached version